docs/workflows.md

```diff
@@ -0,0 +1,15 @@
+CLIMS uses the Camunda Workflow Engine under the hood, which in turn uses [BPMN conventions](https://camunda.com/bpmn/).
```

Comment:
I wonder if "uses BPMN conventions" should instead be "implements the BPMN standard" and include a link to https://www.omg.org/spec/BPMN/2.0/ too.

Reply:
Fixed